server/src/main/java/org/apache/druid/query/lookup/LookupReferencesManager.java

```diff
@@ -373,11 +374,28 @@ private void takeSnapshot(Map<String, LookupExtractorFactoryContainer> lookupMap
     }
   }
 
-  private void loadAllLookupsAndInitStateRef()
+  /**
+   * Load a set of lookups.
+   * @param lookupsToLoad List of lookup names to load. Pass null to load all lookups.
+   */
+  private void loadLookupsAndInitStateRef(List<String> lookupsToLoad)
```

Comment:
Argument not needed as the config is directly accessible from this method itself.
```suggestion
  private void loadLookupsAndInitStateRef()
```